app/[locale]/academics/departments/[name]/page.tsx

```diff
@@ -21,12 +35,122 @@ export default async function Department({
   const department = await db.query.departments.findFirst({
     where: (departments, { eq }) => eq(departments.urlName, name),
   });
-  if (!department) notFound(); // FIXME: Remove this once dynamicParams works
+  // if (!department) notFound(); // FIXME: Remove this once dynamicParams works
+
+  const dummyDepartment = {
+    name: name.toUpperCase(),
+    titleImage:
+      'https://www.figma.com/file/2UXH7QR1dUT8KXgdGuYaC1/image/cf46d1812e6285efda1f8172b7fc3634c6d76130',
+    defaultText:
+      'Lorem ipsum dolor sit amet consectetur adipisicing elit. Consequuntur vero error totam sapiente quidem ad eligendi deserunt perspiciatis ratione dolores corrupti ut sequi neque repellat aspernatur, quos nobis fugit. Assumenda corrupti et qui quaerat consectetur, ex minus vero minima impedit placeat facere, labore iste consequatur ullam accusantium itaque repellat? Necessitatibus?',
```

Comment:
`titleImage`, `defaultText`? Please don't make up imaginary fields, this will make it difficult to port this code over to use the database. Also, all values are already populated for the CS department, you don't need this object at all.

Reply:
so should I just remove them or change there names?

Reply:
Read the last sentence:

> you don't need this object at all.

You can directly fetch from the database instead of making dummy objects.